### ISSUE TITLE
[feature] Early hints support ?

### ISSUE THREAD

**Issue by Clemv95** [ZoeyVid/NPMplus#1263]
Does early hints are supported ?

**Comment by Zoey2936** [comment]
no since neither nginx/freenginx/angie supprts them yeet: https://github.com/nginx/nginx/pull/326 https://github.com/freenginx/nginx/issues/3 https://github.com/webserver-llc/angie/issues/61

**Comment by Clemv95** [comment]
It seems to be on pretty good way for nginx, as the PR is very recent

**Comment by Zoey2936** [comment]
it is still only a Proof of concept

**Comment by Clemv95** [comment]
Of course, but maybe in few weeks it will become stable
Early hints are a pretty nice feature to speed up websites

**Comment by Zoey2936** [comment]
yes this feature will be added, but first it needs to be released by the freenginx/nginx/angie

**Comment by Zoey2936** [comment]
and if it is only in nginx, I'm not sure if I want to switch back to nginx, I would prefer waiting untill it is included in freenginx or angie

**Comment by Zoey2936** [comment]
nginx v1.29.0 was released a few hours ago with early hints support, it will be included in the next NPMplus, release, but I will wait until OpenSSL 3.5.1 is released and available in alpine stable, since nginx also added support for the native openssl quic api, which has a bug which will be fixed in openssl 3.5.1 and yes...

**Comment by Zoey2936** [comment]
will, close as develop now includes early hints support, still currently not recommended to use
